SECURITY.md

```diff
@@ -0,0 +1,43 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `standard`
```

Comment:
By the `standard` project, do we mean our `OBS` instance? I'm not so sure I understand this.